EIPS/eip-slot-precompile.md

```diff
@@ -0,0 +1,60 @@
+---
+eip:
+title: SLOT precompile
+description: Precompile to get the current slot
+author: Marc Harvey-Hill (@Marchhill)
+discussions-to: 
```

Comment:
Please create a discussions topic in Eth Magicians using the template: https://ethereum-magicians.org/c/eips/5